src/components/Header/UserMenu.js

```diff
@@ -46,7 +46,7 @@ const UserMenu = () => {
           aria-labelledby="user-menu"
         >
           <Link
-            href="/"
+            href="/profile"
```

Comment:
This should be `to` not `href`.

Reply:
> Nice work, we just need to iron out a few details.
> I would recommend rebasing this branch as it seems to be a bit out of date, it will also solve the conflict issue. Let me know if you need help with that

I made a rebase using `git rebase nyala-50/edit-profile` from the master branch, is that correct? I see it doesn't resolve the conflict

Reply:
From this branch rebase to master `git rebase master` or `git rebase origin/master`. If it is a hassle and you prefer to resolve the conflict in another way then go for it.

Reply:
@abolkog I did the rebasing..I think it is ready now